jetstream/experimenter.py

```diff
@@ -51,8 +50,8 @@ class Experiment:
     normandy_slug: Optional[str]
     type: str
     status: Optional[str]
-    features: List[str]
     branches: List[Branch]
+    probe_sets: List[str]
```

Comment:
Or should we keep using `features` here?

Reply:
renaming things to `probe_sets` is probably the right call.